config/config.go

```diff
@@ -15,13 +18,16 @@ type Config struct {
 
 	// cache
 	chains Chains `yaml:"-" json:"-"`
+
+	ConfigPath string `yaml:"-" json:"-"`
 }
 
-func DefaultConfig() Config {
+func DefaultConfig(configPath string) Config {
 	return Config{
-		Global: newDefaultGlobalConfig(),
-		Chains: []core.ChainProverConfig{},
-		Paths:  core.Paths{},
+		Global:     newDefaultGlobalConfig(),
+		Chains:     []core.ChainProverConfig{},
+		Paths:      core.Paths{},
+		ConfigPath: configPath,
 	}
 }
```

Comment:
This function can be unexported, right?

Reply:
https://github.com/hyperledger-labs/yui-relayer/commit/286bfe5359804373830a090058994c3dbbb0447b
Rename func